src/herder/Upgrades.cpp

```diff
@@ -69,18 +69,13 @@ Upgrades::isValid(uint64_t closeTime, UpgradeType const& upgrade,
     case LEDGER_UPGRADE_BASE_FEE:
     {
         uint32 newFee = lupgrade.newBaseFee();
-        // allow fee to move within a 2x distance from the one we have in our
-        // config
-        res = (newFee >= mCfg.DESIRED_BASE_FEE * .5) &&
-              (newFee <= mCfg.DESIRED_BASE_FEE * 2);
+        res = (newFee == mCfg.DESIRED_BASE_FEE);
```

Comment:
you're missing another change in this PR: moving to using `timeForUpgrade` for all changes. Right now it's only enforced for `LEDGER_PROTOCOL_VERSION`, which means that nodes would upgrade fees etc before upgrading the protocol version.

Reply:
this may also simplify some tests depending on the sequencing of commits in the PR

Reply:
There was different issue for that one, so I made separate PR with that.